tests/login.test.js

```diff
@@ -0,0 +1,52 @@
+import LoginPage from '../pages/login.page';
+const LOGIN_PAGE = LoginPage;
```

Comment:
This is not needed, as LoginPage is already an instantiated object here, you can directly use it.